randovania/games/fusion/logic_database/Sector 6 NOC.txt

```diff
@@ -930,8 +930,14 @@ Extra - room_id: [27]
   * L0 Hatch to Big Shell 2/Door to Big Shell 1
   * Extra - door_idx: (56,)
   > Door to Shell Access
-      # TODO: Add jump extend and single wall jump template options after confirmation. 98% sure this is doable. -Herc.
-      Space Jump and Can Use Power Bombs
+      All of the following:
+          Can Use Power Bombs
+          Any of the following:
+              Space Jump
+              # Possible with Single Wall Jumping Physics going up the left side.
+              Wall Jump (Intermediate) and Can Single Walljump
+              # Possible without Single Wall Jumping physics by jumping off both walls. Height gain per jump is small.
+              Wall Jump (Advanced)
```

Comment:
Disregard my previous thought; you can easily do this without jump extends. I was just being bad at the game, sorry about that.
Still might be worth bump this wall jump down to intermediate/beginner with Hi-Jump though.

Reply:
Is t his easier with jump extends? might be worth making something like
ANY:
- WJ advanced
- ALL
  - WJ Intermediate
  - ANY
    - HJ
    - Jump extends

Reply:
eh, not really, if anything I find it harder to do wall jumps when I jump extend, don't think it should affect any of the logic here